hack/vendor.sh

```diff
@@ -19,7 +19,12 @@ case $# in
 	;;
 # If user passed arguments to the script
 1)
-	eval "$(grep -E "^clone [^ ]+ $1" "$0")"
+	path=$(pwd)"/hack/vendor.sh"
+	if ! cloneGrep="$(grep -E "^clone [^ ]+ $1" $path)"; then
+    	echo >&2 "error: failed to find 'clone ... $1' in $0"
+    	exit 1
```

Comment:
I updated the pr. PTAL.
Sorry for my not offering the operating system info. I think it bothers you a lot.
Thanks for your patience.